src/typed-method-types/workflows/triggers/tests/event_test.ts

```diff
@@ -85,13 +85,10 @@ Deno.test("Mock call for event", async (t) => {
             },
           });
           assertEquals(res.ok, true);
-          if (res.ok) {
```

Comment:
The assertion above this line should ensure `res.ok` is always true. As a result, this conditional is not needed. (similar change made to other similar tests in other files below)